### ISSUE TITLE
remove `files/`from the start of file fetches

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#2008]
from our docs, under the heading "uploading data to an iodide notebook" -- https://iodide-project.github.io/docs/workflows/#getting-data-into-an-iodide-notebook

having now spent some time with the syntax
```
%% fetch
TYPE: VAR_NAME = files/FILE_NAME
```

it's now feeling like extra boilerplate, and i don't think it clarifies the mental model for our users. i think we should deprecate the "files/" bit, and just use
```
%% fetch
TYPE: VAR_NAME = FILE_NAME
```

so that this seems like a normal local variable reference.

@hamilton what do you think?

**Comment by bcolloran** [comment]
also, this would make the fetch cell consistent with the file api, which does not use the `files/` prefix -- https://iodide-project.github.io/docs/api/#iodidefile

**Comment by hamilton** [comment]
Good points, but I think it might be good to have a discussion about this especially before #1919 gets merged. The original idea behind `files/` was that if we end up eventually exporting an archivable bundle, we put the notebook's associated files in the resulting `files/` directory, so in theory a static version of the notebook would effortlessly be able to fetch the files in `files/` without the server. It does seem like we're giving up on having any form of that model, so perhaps this won't matter as much going forward.

**Comment by bcolloran** [comment]
as is, the fetch cell already intercepts calls to load local files and processes them specially, so we can still put files in a `files/` folder and rewrite the fetch request if we need to. i think the question about the end user experience is orthogonal to the implementation details behind the scenes, and it feels weird to expose an implementation detail for a feature that we had proposed but never implemented, especially when it's inconsistent with the choices we made for the file api.

**Comment by bcolloran** [comment]
fixed via #1919
